packages/ghauri/PKGBUILD

```diff
@@ -9,27 +9,42 @@ arch=('any')
 groups=('blackarch' 'blackarch-webapp' 'blackarch-exploitation')
 url='https://github.com/r0oth3x49/ghauri'
 license=('MIT')
-depends=('python' 'python-chardet' 'python-tldextract' 'python-colorama'
-         'python-requests')
-makedepends=('git' 'python-setuptools')
+depends=('python')
```

Comment:
@s7x thx mate :) if there is no version bump then `pkgrel++` i.e. `pkgrel=2` please.